src/Avalonia.Base/Threading/DispatcherImplExceptionWrapper.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading;
+
+namespace Avalonia.Threading;
+
+internal class DispatcherImplExceptionWrapper : IControlledDispatcherImpl
```

Comment:
I won't review this feature much, leaving it to @kekekeks.
But why is it in IControlledDispatcherImpl instead of Dispatcher itself?

In general, there are at least these points where we should handle unhandled exceptions:
- Controlled dispatcher loop that runs DispatcherFrame
- Dispatcher jobs that are running on a timer (browser, mobile, where there is no controlled loop)
- Double-check that all input exceptions can be handled by the dispatcher as well (previously it wasn't the case, and exception in PointerMoved could miss dispatcher processing completely).
- Other platform events that might skip Dispatcher processing. Some platform callbacks like BackButton events, maybe? It might be worth redirecting these to the dispatcher queue.

We also have rendering exceptions on Render thread, but these are logged right now... Possibly, we can add a global handler there as well.

Reply:
There is also another problem, that some Avalonia code might not gracefully recover in case of any exceptions. Like, missing finally blocks in random places... But these can be found later.